Tema 2/exportToHTML/Exercitiul1.java.html

```diff
@@ -0,0 +1,45 @@
+<html>
```

Comment:
when I said write javadoc, I didn't mean generate the html.
Please don't generate it in the future, it makes the pull request a bit more crowded without extra information